components/Table/src/stories/react.stories.mdx

```diff
@@ -0,0 +1,211 @@
+import { Canvas, Description, Meta, Story } from "@storybook/addon-docs";
+
+import { Table, TableBody, TableCaption, TableCell, TableHead, TableHeader, TableRow } from "../index";
+import pkg from "../../package.json";
+import readme from "../../README.md";
+
+<Meta
+  title="React Components/Data Display/Table"
+  component={Table}
+  parameters={{
+    componentSubtitle: `${pkg.name} - ${pkg.version}`,
+    docs: { source: { type: "dynamic" } },
+    status: {
+      type: "WORK IN PROGRESS",
+    },
+  }}
+/>
+
+<Description markdown={readme} />
+
+## Stories
+
+### Default
+
+<Canvas>
+  <Story name="Default">
+    {() => (
+      <div aria-labelledby="caption_default" role="group">
+        <Table>
+          <TableCaption id="caption_default">He-Man and Skeletor facts</TableCaption>
+          <TableHead>
+            <TableRow>
+              <TableCell></TableCell>
+              <TableHeader className="heman">He-Man</TableHeader>
+              <TableHeader className="skeletor">Skeletor</TableHeader>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <TableHeader scope="row">Role</TableHeader>
+              <TableCell data-heading="He-Man">Hero</TableCell>
+              <TableCell data-heading="Skeletor">Villain</TableCell>
+            </TableRow>
+            <TableRow>
+              <TableHeader scope="row">Weapon</TableHeader>
+              <TableCell data-heading="He-Man">Power Sword</TableCell>
+              <TableCell data-heading="Skeletor">Havoc Staff</TableCell>
+            </TableRow>
+            <TableRow>
+              <TableHeader scope="row">Dark secret</TableHeader>
+              <TableCell data-heading="He-Man">Expert florist</TableCell>
+              <TableCell data-heading="Skeletor">Cries at romcoms</TableCell>
+            </TableRow>
+          </TableBody>
+        </Table>
+      </div>
+    )}
+  </Story>
+</Canvas>
+
+### Hover
+
+<Canvas>
+  <Story name="Hover">
+    {() => (
+      <div aria-labelledby="caption_hover" role="group">
+        <Table>
+          <TableCaption id="caption_hover">He-Man and Skeletor facts</TableCaption>
+          <TableHead>
+            <TableRow tabIndex="-1">
+              <TableCell></TableCell>
+              <TableHeader className="heman">He-Man</TableHeader>
+              <TableHeader className="skeletor">Skeletor</TableHeader>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            <TableRow>
+              <TableHeader scope="row">Role</TableHeader>
+              <TableCell data-heading="He-Man">Hero</TableCell>
```

Comment:
What's the exact reasoning behind the `data-heading` attributes?

Reply:
@paulAcato correct me if I'm missing something. From what I understood, the `data-heading` attribute is used in the CSS to give some flexibility in creating a different layout for mobile. On mobile display sizes, a title is placed before each cell using the `attr()` css function. 

The advantage is that it allows us to support a completely different table layout for mobile (vertical, as per [our design](https://www.figma.com/file/JpoY3waVoQGlLQzQXTL9nn/Design-System---Gemeente-Den-Haag?node-id=5213%3A16235)), without creating a whole new component. 

A few disadvantages I can think of:
- it forces you to add the heading attribute to every cell, this could be very tedious for tables that are hand-made;
- there's a risk of inconsistent titles if a developer forgets to update the `data-heading` when the table header is changed;
- we're introducing complexity in using the BEM classes (although it's optional);
- semantically, a vertical table is very similar to a [description list](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl). It might be better to just create a description list component and leave it to developers to swap them accordingly.

Reply:
@Robbert any thoughts on this?

Reply:
@bddjong I think during the refinements I made similar points, I still have to see the first accessible responsive table. My preference was too to provide an alternative definition list for specific cases. If I recall correctly, optimism about the feasibility and challenges in WordPress were reasons to try this approach first. But I'm not convinced yet :)